### DISCUSSION TITLE
win screen - end of game

### DISCUSSION THREAD

**Discussion by mya1919** [jeffaroundtheworld/2024gamecollab#85] [Q&A]
how do we feel about this being the outside of the school, so if the player wins and "esapes" the school it shows this as the end as like the "win screen." (we could do a few photos rotating like the start screen)
![Screenshot 2024-04-30 145438](https://github.com/jeffaroundtheworld/2024gamecollab/assets/159198555/1693003c-96c7-40f0-9f3a-4f96919dc538)
![Screenshot 2024-04-30 145546](https://github.com/jeffaroundtheworld/2024gamecollab/assets/159198555/a817aa03-1233-45ff-a131-438027c48be2)

**Comment by kodriq** [comment]
that would be good
